### DISCUSSION TITLE
How to sync saves in Dual boot setup with Lutris and Ludusavi

### DISCUSSION THREAD

**Discussion by nikhilCad** [mtkennerly/ludusavi#270] [Q&A] [answered]
Hello, I am new to Linux in general. I have got my games running on Lutris, I was already using Ludusavi on Windows and have my games saves backed up from there. I want to ask how can I restore those save files in my Linux dual boot to use in Lutris?

**Comment by mtkennerly** [comment, marked as answer]
Hi! Syncing saves across OSes is very difficult. Essentially, each Windows game on Linux would have a Wine prefix folder that's an imitation of a C drive, so you'd have to copy the save files into the Wine prefix at the equivalent of their normal Windows location. It's possible to do that with Ludusavi's redirects feature, but it's quite tedious. I would recommend watching https://github.com/mtkennerly/ludusavi/issues/194 for any futures updates where Ludusavi could help streamline this.
